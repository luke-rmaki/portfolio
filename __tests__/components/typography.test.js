import React from 'react';
import renderer from 'react-test-renderer';

import {
  MainHeading,
  Paragraph,
  Subheading,
} from '../../src/components/typography';

describe('MainHeading', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<MainHeading />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('Paragraph', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Paragraph />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('Subheading', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Subheading />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
