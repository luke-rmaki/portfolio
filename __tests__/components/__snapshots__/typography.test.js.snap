// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainHeading renders correctly 1`] = `
<h1
  className="sc-bdVaJa cukzsS"
/>
`;

exports[`Paragraph renders correctly 1`] = `
<p
  className="sc-htpNat fVEbsh"
/>
`;

exports[`Subheading renders correctly 1`] = `
<h2
  className="sc-bwzfXH hGAVIm"
/>
`;
